### PR TITLE
Add interval to chart to make in configurable on deployment

### PR DESCRIPTION
**Pull request by colmm99**
#### What this PR does / why we need it:
The default is 1m and this has caused some issues with AWS route53 ratelimit.
This option give you the capability to set the interval

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/external-dns/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ description:
   Configure external DNS servers (AWS Route53, Google CloudDNS and others)
   for Kubernetes Ingresses and Services
 name: external-dns
-version: 1.3.3
+version: 1.3.4
```
Comment (cpanato) [inline stable/external-dns/Chart.yaml]: adding new funcionality 
```suggestion
version: 1.4.0
```

File: stable/external-dns/README.md [hunk 2]
```diff
@@ -68,6 +68,7 @@ The following table lists the configurable parameters of the external-dns chart
 | `publishInternalServices`          | Allow external-dns to publish DNS records for ClusterIP services (optional).                                               | `false`                                            |
 | `rbac.create`                      | If true, create & use RBAC resources                                                                                       | `false`                                            |
 | `rbac.serviceAccountName`          | Existing ServiceAccount to use (ignored if rbac.create=true)                                                               | `default`                                          |
+| `interval`                         | Interval update period to use (options: txt, noop)                                                                                | `txt`                                              |
```
Comment (yurrriq) [inline stable/external-dns/README.md:71]: This looks like a copy/paste error. As per `values.yaml`, the default is `1m`, and the value should be a time duration, so `options: txt, noop` is wrong too.

**Review by yurrriq — COMMENTED** [review]
`README.md` needs to be updated.

**Pull request merged**